src/utils/votes.ts

```diff
@@ -0,0 +1,9 @@
+import { YesNoVote } from '@votingworks/ballot-encoder'
+import { YesNo } from '../config/types'
```

Comment:
To make this even more clear…
```suggestion
import { YesOrNo } from '../config/types'
```

Reply:
Fixed.